src/components/PatchesPage/PatchCard.tsx

```diff
@@ -106,7 +109,10 @@ export const PatchCard: React.VFC<Props> = ({
       </Left>
       <Center>
         <PatchBadgeContainer>
-          <PatchStatusBadge status={versionFull?.status ?? status} />
+          {!isUnconfigured && (
+            <PatchStatusBadge status={versionFull?.status ?? status} />
```

Comment:
I think some change here broke the alignment
Before:
<img width="948" alt="image" src="https://github.com/evergreen-ci/spruce/assets/4605522/8a196ffa-56e9-4825-bfe7-2c8e9831f0ca">

After:
<img width="423" alt="image" src="https://github.com/evergreen-ci/spruce/assets/4605522/dc734b6b-8fed-430b-b4b4-af4a1b8f4fec">